f1-telemetry/src/packet/session.rs

```diff
@@ -109,6 +109,11 @@ pub enum Formula {
     F1Classic,
     F2,
     F1Generic,
+    Beta,
+    Supercars,
+    Esports,
+    F221,
```

Comment:
Could you rename to `F2_21`? I just think it's hard to read without the underscore.